src/Umbraco.Web.BackOffice/Trees/MemberTreeController.cs

```diff
@@ -64,6 +64,7 @@ public class MemberTreeController : TreeController, ISearchableTree, ITreeNodeCo
         /// <returns></returns>
         public ActionResult<TreeNode> GetTreeNode(string id, [ModelBinder(typeof(HttpQueryStringModelBinder))]FormCollection queryStrings)
         {
+            //TODO: this is currently throwing an exception when loading a member
```

Comment:
@Shazwazza @bergmania the id being passed in is currently null. I'm trying to debug to find out why. The member does load, but an '_Object reference not set to an instance of an object._' exception is thrown first as **id** is null. 

It seems like the `/umbraco/backoffice/umbracotrees/membertree/GetTreeNode/[id]` call isn't working as expected, yet I can see the link being generated. Will need to look into this more unless you have any ideas?

Reply:
FYI I just replicated this on the default `netcore` branch so this will need a new bug to be created, but is not as part of this PR. cc @bergmania

Reply:
Fixed by @bergmania, added `[FromRoute]`, to stop the ID being empty, many thanks